### PR TITLE
Centrifuge App: Token list page MVPv1

### LINKED ISSUE #732: Token list / overview page - MVPv1
[Specs](https://centrifuge.hackmd.io/m4QFRZzoQs-h7P7qcMzC_w)
[Figma frame - Token overview/list](https://www.figma.com/file/LCbVWH8fXPkXMbLpC3VZtx/Centrifuge-app---Altair?node-id=3692%3A83628)

- A user can view tokens
- A user does not have the be an investor, it can be a guest user.

When a token is selected the user redirects to the token details page
The list can be sorted by values (most to least and visa versa, for **Yield**, **Protection** and **Capacity**)

- Show list of live tokens
    - Displaying their **Token Name and Icon**
        - Selected during tranche creation; [Specs](https://centrifuge.hackmd.io/1OiMlCCWTL2YwUN8c_sFnA#Tranches)
    - **Asset class**
        - Selected during pool creation; [Specs](https://centrifuge.hackmd.io/1OiMlCCWTL2YwUN8c_sFnA#Pool-details)
    - Values for each token
        - **Yield**
            - 30D for Senior; [calculation](https://docs.centrifuge.io/learn/terms/#30d-drop-yield)
            - 90D for Junior; [Calculation](https://docs.centrifuge.io/learn/terms/#90d-tin-yield)
            
        - **Protection**
            - Display the actual protection
                - [See this document on risk and protectionlevel details and calculations](https://docs.centrifuge.io/learn/terms/#tin-risk-buffer)
        - **Capacity**
            - How much Capacity is available for investment.
                -  Spec; [How investment capacity is calcualted](https://centrifuge.hackmd.io/B-i5X31FSdSOglkh1J-vVg?view#How-investment-capacity-is-calculated)

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### 📚 Description

This pull request adds the token overview page `/tokens` (see design in [Figma](https://www.figma.com/file/d1bEBkwllUQe27nQM6VlJu/Centrifuge-app?node-id=3692%3A83628)).

Decisions made:
1. Calculation will be updated at a later point in time since we need to access historical data for them
2. I'm assuming in prod that tranches and pools will _always_ have names and asset classes defined - in our test data this is not necessarily the case
3. developed in light mode for now
4. ~design is ever changing, will have to touch/finalize the page again when design is finished. For now the page was built with the specs from before April 5th ([this figma link](https://www.figma.com/file/d1bEBkwllUQe27nQM6VlJu/Centrifuge-app?version-id=1729022115&node-id=3434%3A89089) should be the version of the page I built)~

Closes #732

### ✅ Approvals

- [ ] Dev

### 📸 Screenshots

Screenshot 2022-04-05 at 10 47 47

### 🌎 Impact

This page should closely resemble the design in figma. Please note that the page is in active design mode so some things may have changed since...

### CODE REVIEW

File: nft-studio/src/components/TokenList.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        .flat(),
+    [pools, utils]
+  )
+
+  return (
+    <DataTable
+      data={tokens}
+      columns={columns}
+      onRowClicked={(token: Row) => {
+        history.push(`/tokens/${token.name}`)
```
Comment (sophialittlejohn) [inline nft-studio/src/components/TokenList.tsx:85]: the path is temporary, will think update it when working on the token detail page

File: nft-studio/package.json [hunk 2]
```diff
@@ -64,7 +64,8 @@
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-react": "7.21.5",
     "netlify-cli": "^6.14.5",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
+    "tsc": "^2.0.4"
```
Comment (sophialittlejohn) [inline nft-studio/package.json:68]: just added this so we don't have to have it installed locally

File: nft-studio/src/components/PageSummary.tsx [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
+    >
+      {data?.map(({ label, value }, index) => (
+        <Stack gap="4px" key={`${value}-${label}-${index}`}>
+          <Text variant="body2" underline>
+            {/* // TODO: placeholder for tooltip */}
+            {label}
+          </Text>
+          <Text variant="heading3">{value}</Text>
+        </Stack>
+      )) || { children }}
```
Comment (onnovisser) [inline nft-studio/src/components/PageSummary.tsx:31]: Nice. Only needs to be just `children` instead of `{ children }`

**Review by onnovisser — APPROVED** [review]
Nice! 💯

**Pull request merged**